app/src/androidTest/java/mozilla/lockbox/Navigator.kt

```diff
@@ -108,6 +109,23 @@ class Navigator {
         fingerprintOnboardingScreen { exists() }
     }
 
+    fun gotoAutofillOnboarding(goManually: Boolean = false) {
```

Comment:
Adding `goManually` to all these is overkill; I can't think of a case when we would care to differentiate. Just have them all go through the `gotoFxALogin()` and placeholder login taps.

Reply:
Removed those for fingerprint and autofill